client/src/snippets.json

```diff
@@ -125,14 +125,14 @@
     "Data parameter with single required dataset": {
         "prefix": "gx-data-single-param",
         "body": [
-            "<param argument=\"--$1\" type=\"data\" format=\"$2\" label=\"$3\" help=\"${4:TODO}\" />"
+            "<param name=\"$1\" argument=\"--$2\" type=\"data\" format=\"$3\" label=\"$4\" help=\"${5:TODO}\" />"
         ],
         "description": "Data parameter with single required dataset"
     },
     "Data parameter with multiple required datasets": {
         "prefix": "gx-data-multiple-param",
         "body": [
-            "<param argument=\"--$1\" type=\"data\" format=\"$2\" multiple=\"true\" label=\"$3\" help=\"${4:TODO}\" />"
+            "<param name=\"$1\" argument=\"--$2\" type=\"data\" format=\"$3\" multiple=\"true\" label=\"$4\" help=\"${5:TODO}\" />"
```

Comment:
name is not needed when we have argument, most of the times ... and I think IUC recommends to order argument before name

Reply:
Yes but usually data inputs have not arguments at all, then I think that at least in those cases is still useful.

Reply:
I modified the order of the parameter. If you think that the name field is not necessary I can remove it.

Reply:
ping @bgruening

Reply:
The name parameters were removed.